app/assets/javascripts/discourse/app/components/topic-map/topic-map-summary.gjs

```diff
@@ -1,162 +1,443 @@
+                        <span class="like-section__user">
+                          {{avatar
+                            post.avatar_template
+                            "tiny"
+                            (hash title=post.username)
+                          }}
+                          {{post.username}}
+                        </span>
+                        <span class="like-section__likes">
+                          {{post.like_count}}
+                          {{dIcon "heart"}}</span>
+                        <p>
+                          {{htmlSafe (emojiUnescape post.blurb)}}
+                        </p>
+                      </a>
+                    </li>
+                  {{/each}}
+                </ul>
+              </ConditionalLoadingSpinner>
+            </section>
+          </:content>
+        </DMenu>
       {{/if}}
-      {{#if (gt @topic.like_count 0)}}
-        <li class="secondary likes">
-          {{number @topic.like_count noTitle="true"}}
-          <h4 role="presentation">{{i18n
-              "likes_lowercase"
-              count=@topic.like_count
-            }}</h4>
-        </li>
+
+      {{#if this.linksCount}}
+        <DMenu
+          @arrow={{true}}
+          @identifier="map-links"
+          @interactive={{true}}
+          @triggers="click"
+          @modalForMobile={{true}}
+          @groupIdentifier="topic-map"
+          @placement="right"
+          @inline={{true}}
+        >
+          <:trigger>
+            {{number this.linksCount noTitle="true"}}
+            <span role="presentation">{{i18n
+                "links_lowercase"
+                count=this.linksCount
+              }}</span>
+          </:trigger>
+          <:content>
+            <section class="links">
+              <h3>{{i18n "topic_map.links_title"}}</h3>
+              <table class="topic-links">
+                <tbody>
+                  {{#each this.linksToShow as |link|}}
+                    <tr>
+                      <td>
+                        <span
+                          class="badge badge-notification clicks"
+                          title={{i18n "topic_map.clicks" count=link.clicks}}
+                        >
+                          {{link.clicks}}
+                        </span>
+                      </td>
+                      <td>
+                        <TopicMapLink
+                          @attachment={{link.attachment}}
+                          @title={{link.title}}
+                          @rootDomain={{link.root_domain}}
+                          @url={{link.url}}
+                          @userId={{link.user_id}}
+                        />
+                      </td>
+                    </tr>
+                  {{/each}}
+                </tbody>
+              </table>
+              {{#if
+                (and
+                  (not this.allLinksShown)
+                  (lt TRUNCATED_LINKS_LIMIT this.topicLinks.length)
+                )
```

Comment:
Might be better to have a `hasMoreLinks` (or `shouldShowAllLinks`) and keep these conditions in the `class` rather than the template?